### PR TITLE
scripts: profiler: Documentation update

### PR DESCRIPTION
**Pull request by MarekPieta**
Change updates command line arguments described in documentation (to match scripts).
Makes script for merging data from two devices more generic and adds documentation.

### CODE REVIEW

**Comment by MarekPieta** [comment]
@pdunaj could you review ?

File: include/profiler.rst [hunk 1]
```diff
@@ -113,6 +113,12 @@ To use the tools, run the scripts on the command line:
   Connects to the device via RTT, plots data in real time, and saves the data.
   As command line arguments, provide a dataset name.
 
+* ``python3 merge_data.py test_p sync_event_p test_c sync_event_c test_merged``
+
+  Combines data from test_p and test_c datasets into one dataset (test_merged).
+  Provides clock drift compensation based on synchronization events: sync_event_p and sync_event_c.
+  This enables you to observe times between events for the two connected devices.
+  As command line arguments, provide the following datasets: names of events used for synchronization for a Peripheral (sync_event_p) and a Central (sync_event_c), names of the Peripheral (test_p) and the Central (test_c), and the merge result (test_merged).
```
Comment (MarekPieta) [inline include/profiler.rst]: @b-gent 
Can names of events be marked as datasets?
Only test_p, test_c, test_merged are names of collected datasets.

Comment (MarekPieta) [reply include/profiler.rst]: Maybe `... provide names of events used for synchronization for a Peripheral (sync_event_p) and a Central (sync_event_c), and names of datasets for: Peripheral (test_p) and the Central (test_c), and the merge result (test_merged).`
If it's ok for you I will make the change

Comment (b-gent) [reply include/profiler.rst]:
```suggestion
  As command line arguments, provide names of events used for synchronization for a Peripheral (sync_event_p) and a Central (sync_event_c), as well as names of datasets for: the Peripheral (test_p), the Central (test_c), and the merge result (test_merged).
```

File: include/profiler.rst [hunk 2]
```diff
@@ -99,20 +99,26 @@ Set :option:`CONFIG_PROFILER_NORDIC` to enable this backend.
...
   Connects to the device via RTT, plots data in real time, and saves the data.
-  As command line arguments, provide the files where to save the data.
+  As command line arguments, provide a dataset name.
 
+* ``python3 merge_data.py test_p sync_event_p test_c sync_event_c test_merged``
+
+  Combines data from test_p and test_c datasets into one dataset (test_merged).
+  Provides clock drift compensation based on synchronization events: sync_event_p and sync_event_c.
+  This enables you to observe times between events for the two connected devices.
+  As command line arguments, provide names of events used for synchronization for a Peripheral (sync_event_p) and a Central (sync_event_c), as well as names of datasets for: the Peripheral (test_p), the Central (test_c), and the merge result (test_merged).
```
Comment (greg-fer) [inline include/profiler.rst]: Is this line complementing `This enables you to observe times between events for the two connected devices.`?
If not, can you add a blank linke between lines 120 and 121?

File: scripts/profiler/merge_data.py [hunk 3]
```diff
@@ -9,40 +9,48 @@
 
 
 def main():
-    descr = "Merge data from device and dongle. ble_peer_events should be" \
-            " registered at the beginning and at the end of measurements"  \
-            " (used to compensate clock drift)."
+    descr = "Merge data from peripheral and central. Synchronization events" \
```
Comment (greg-fer) [inline scripts/profiler/merge_data.py]: Capitalization inconsistency with profiler.rst's line 121.

File: include/profiler.rst (hunk 2 above)
Comment (MarekPieta) [reply include/profiler.rst]: Clock drift compensations make us able to properly observe times between events. So i think this line is complementing previous one,

File: scripts/profiler/merge_data.py (hunk 3 above)
Comment (MarekPieta) [reply scripts/profiler/merge_data.py]: ok, I will create new PR for this one (since this one is already closed)

**Pull request merged**